mobile/src/components/chess/Board.tsx

```diff
@@ -66,6 +72,11 @@ const Board = ({ color }) => {
       chess.load(fen);
       onTurn();
     });
+
+    // reset state when component unmounts
+    return () => {
+      setState(initChessState);
```

Comment:
This change fixes the following error:
![image](https://user-images.githubusercontent.com/64985802/169716959-a2010a2c-3d26-408a-aea0-2f1af15cbae6.png)

Reply:
Can you offer an explanation about this? Very curious.

Reply:
References
- https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html
- https://stackoverflow.com/questions/53949393/cant-perform-a-react-state-update-on-an-unmounted-component